__tests__/app.test.js

```diff
@@ -71,6 +71,7 @@ describe("GET", () => {
               topic: "mitch",
               created_at: "2020-11-03T09:12:00.000Z",
```

Comment:
I wouldn't update old tests. this should be a new test. does it also include comment_count?